pytorch/inference/docker/1.11/py3/Dockerfile.cpu

```diff
@@ -149,9 +149,11 @@ RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pytho
  && pip uninstall -y torch torchvision torchaudio torchdata \
  && pip install --no-cache-dir -U ${PT_E3_INFERENCE_URL} ${PT_TORCHVISION_URL} ${PT_TORCHAUDIO_URL} ${PT_TORCHDATA_URL}
```

Comment:
We build e3 images using `PT_E3_INFERENCE_URL`. Thereafter, we build SM on top of E3 images. Which means that SM images are using `PT_E3_INFERENCE_URL`. Is that intended?

nit: If yes, in future we should give `PT_E3_INFERENCE_URL` a more generic name such as `PT_INFERENCE_URL`

Reply:
Agreed.